rocks-bin/src/list.rs

```diff
@@ -0,0 +1,43 @@
+use clap::Args;
+use eyre::{OptionExt as _, Result};
+use itertools::Itertools as _;
+use rocks_lib::{config::Config, tree::Tree};
+use text_trees::{FormatCharacters, StringTreeNode, TreeFormatting};
+
+#[derive(Args)]
+pub struct List {
```

Comment:
suggestion: name this `ListCmd` to clarify that this isn't a list. 
Clap has macros that let you specify the name on the CLI

Reply:
Yeah good call, I'll change that.